### PR TITLE
Add "Speed up charge bar during one-foe attacks" of masterBonus

### PR DESCRIPTION
**Pull request by kagekiyo7**
"Boost to main weapon's ATK when main weapon is ***" is impossible because motocal does not have a main weapon concept.

Add support for "Speed up charge bar during one-foe attacks". (通常攻撃時奥義ゲージ上昇)

### CODE REVIEW

**Comment by h-yasha** [comment]
@kagekiyo7 could you also add Kengo's Level Bonus?

**Comment by kagekiyo7** [comment]
I can probably do it. After this PR is closed.

**Comment by OrkunKocyigit** [comment]
I am not sure but shouldn't formula should be like this.
expectedOugiGage += (ougiGageBuff + buff["masterBonusOugiGage"]) * (taRate * buff["masterBonusOugiGage"] * 3 + (1.0 - taRate) * (daRate * buff["masterBonusOugiGage"] * 2 + (1.0 - daRate)));

I am assuming of course these buffs are additive to each other instead of multiplier of each other.

**Comment by kagekiyo7** [comment]
Oh, I verified now, ougiGageBuff was not applied.
Is there a problem with this?

**Comment by kagekiyo7** [comment]
Also, this is not a buff. 
According to the number of attacks, ougi gage UP.

TA: ougi gage +3
DA: ougi gage +2
single attack: +1

**Comment by kagekiyo7** [comment]
If there is no problem, I want to merge.

**Comment by kagekiyo7** [comment]
test myself
TA100% (37+3) : ok
image
DA100% (22+2) : ok
image
SA100% (10+1) : ok
image

**Comment by kagekiyo7** [comment]
test myself (if djeeta gained ougiGageBuff.)
DA: 30 + (22 * 1.5) + ((1 + 1) * 2) = 67
(base + (22 * ougiGageBuff) +  ((masterBonus + kengo level bonus) * attacks number)
image

File: src/translate.js [hunk 1]
```diff
@@ -869,11 +869,30 @@ var multiLangData = {
...
     },
     "マスボダメ上限説明": {
-        "en": "The value of Job Master bonus \"DMG Cap+○○%\"",
+        "en": "The value of Job Master bonus \"DMG Cap+**%\"",
         "ja": "ジョブマスターボーナスの\"ダメージ上限+○○％\"の値です\n(各ジョブごとのボーナスとは別です)",
         "zh": "职业满级Bonus中\"DMG Cap+○○％\"中的值\n(和各职业的Bonus是独立的)",
     },
-
+    "マスボ奥義ゲージ": {
+        "en": "Normal Attack Charge Bar",
```
Comment (OrkunKocyigit) [inline src/translate.js]: This English key might be confusing. Can you update this as "One-foe Attacks Bonus Charge" as it is how rephrased in the game?

Comment (kagekiyo7) [reply src/translate.js]: Thank you for the advice. My english skill is low, so I was saved.

**Pull request merged**